recipes/geotrans/all/test_package/conanfile.py

```diff
@@ -0,0 +1,30 @@
+import os
+from conans import ConanFile, CMake, tools
+
+
+class GeotransTestConan(ConanFile):
+    settings = "os", "compiler", "build_type", "arch"
+    generators = (
+        "cmake",
+        "cmake_find_package",
+        "cmake_paths",
+    )
```

Comment:
I don't think `conan_paths` is needed.
```suggestion
    generators = "cmake", "cmake_find_package"
```